ci/workflows/main.yml

```diff
@@ -213,39 +213,6 @@ steps:
     timeout_in_minutes: 40
```

Comment:
Might TLE since it includes more tests now, but there's no way to tell, until after merging.

Reply:
Yeah I will keep an eye on the workflow